autoscaler/controllers/gateway/config/quickwit.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	odigosv1 "github.com/keyval-dev/odigos/api/odigos/v1alpha1"
+	commonconf "github.com/keyval-dev/odigos/autoscaler/controllers/common"
+	"github.com/keyval-dev/odigos/common"
+)
+
+const (
+	qwUrlKey = "QUICKWIT_URL"
+)
+
+type Quickwit struct{}
+
+func (e *Quickwit) DestType() common.DestinationType {
+	return common.QuickwitDestinationType
+}
+
+func (e *Quickwit) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) {
+	if url, exists := dest.Spec.Data[qwUrlKey]; exists {
+		exporterName := "otlp/generic"
```

Comment:
we recently had a bug where multiple destination of the same type would override each other configuration, thus it is recommended to add the destination name to make it unique in the yaml.

You can check out [this pr](https://github.com/keyval-dev/odigos/pull/906/files) for reference

Reply:
Done